Sources/PostgresNIO/New/PSQLData.swift

```diff
@@ -0,0 +1,219 @@
+//
+//  File.swift
+//  
+//
+//  Created by Fabian Fett on 13.01.21.
+//
+
+/// The format code being used for the field.
```

Comment:
Ditch Xcode template headers.
```suggestion
/// The format code being used for the field.
```